### PR TITLE
Add lint and rubocop compliance + CentOS 7 support + remove useless packages

### PR DESCRIPTION
**Pull request by CharlesJUDITH**
This PR will add lint compliance.

---

This change is [Reviewable](https://reviewable.io/reviews/criteo-cookbooks/centreon-engine/3)

### CODE REVIEW

File: recipes/repo.rb [hunk 1]
```diff
@@ -5,17 +5,16 @@
...
+    description node['centreon_engine'][repo]['description']
+    enabled node['centreon_engine'][repo]['enabled']
+    gpgcheck node['centreon_engine'][repo]['gpgcheck']
+    gpgkey node['centreon_engine'][repo]['gpgkey']
+    repositoryid node['centreon_engine'][repo]['repositoryid']
 
-      action :create
-    end
+    action :create
+    only_if { node['centreon_engine'][repo]['managed'] == true }
```
Comment (kamaradclimber) [inline recipes/repo.rb]: you don't have to test for equality with true:

```
only_if { node['centreon_engine'][repo]['managed'] }
```

**Comment by kamaradclimber** [comment]
Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks broke.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3)_

**Comment by jmauro** [comment]
*[attributes/default.rb, line 26 at r2](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3#-KYrbNMUVutwjluK0S6o:-KYrbNMUVutwjluK0S6p:b-pg34jr) ([raw file](https://github.com/criteo-cookbooks/centreon-engine/blob/5fdb12fc0dc7d1527929936dd54695c377791731/attributes/default.rb#L26)):*
> ```Ruby
>   default['centreon_engine']['ces-standard']['repositoryid'] = 'ces-standard'
>   default['centreon_engine']['ces-standard']['description'] = 'Centreon RPM repository for ces $releasever'
>   default['centreon_engine']['ces-standard']['baseurl'] = "http://yum.centreon.com/standard/3.4/el#{release}/stable/$basearch/"
> ```

To be more generic you should use the yum variable: `el$releasever`

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3)*

**Comment by jmauro** [comment]
*[attributes/default.rb, line 29 at r2](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3#-KYrbg2e5d5_Ue-zEyMO:-KYrbg2e5d5_Ue-zEyMP:b-ya81ng) ([raw file](https://github.com/criteo-cookbooks/centreon-engine/blob/5fdb12fc0dc7d1527929936dd54695c377791731/attributes/default.rb#L29)):*
> ```Ruby
>   default['centreon_engine']['ces-standard']['gpgcheck'] = true
>   default['centreon_engine']['ces-standard']['enabled'] = true
>   default['centreon_engine']['ces-standard']['gpgkey'] = "http://yum.centreon.com/standard/3.4/el#{release}/stable/RPM-GPG-KEY-CES"
> ```

Same as above

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3)*

**Comment by jmauro** [comment]
*[attributes/default.rb, line 34 at r2](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3#-KYrbiLo7J3IDCRdfmBP:-KYrbiLpxV--yyMudTIY:b-ya81ng) ([raw file](https://github.com/criteo-cookbooks/centreon-engine/blob/5fdb12fc0dc7d1527929936dd54695c377791731/attributes/default.rb#L34)):*
> ```Ruby
>   default['centreon_engine']['ces-standard-noarch']['repositoryid'] = 'ces-standard-noarch'
>   default['centreon_engine']['ces-standard-noarch']['description'] = 'Centreon RPM repository for ces $releasever'
>   default['centreon_engine']['ces-standard-noarch']['baseurl'] = "http://yum.centreon.com/standard/3.4/el#{release}/stable/noarch/"
> ```

Same as above

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3)*

**Comment by jmauro** [comment]
Reviewed 2 of 4 files at r1, 4 of 4 files at r2.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3)*

**Comment by CharlesJUDITH** [comment]
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---

*[attributes/default.rb, line 26 at r2](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3#-KYrbNMUVutwjluK0S6o:-KYrfAUuV9eBjDqkDenM:b-896fix) ([raw file](https://github.com/criteo-cookbooks/centreon-engine/blob/5fdb12fc0dc7d1527929936dd54695c377791731/attributes/default.rb#L26)):*
<details><summary><i>Previously, jmauro wrote…</i></summary><blockquote>

To be more generic you should use the yum variable: `el$releasever`
</blockquote></details>

Done.

---

*[attributes/default.rb, line 29 at r2](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3#-KYrbg2e5d5_Ue-zEyMO:-KYrfBKHK18Y46Im9-cv:b-896fix) ([raw file](https://github.com/criteo-cookbooks/centreon-engine/blob/5fdb12fc0dc7d1527929936dd54695c377791731/attributes/default.rb#L29)):*
<details><summary><i>Previously, jmauro wrote…</i></summary><blockquote>

Same as above
</blockquote></details>

Done.

---

*[attributes/default.rb, line 34 at r2](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3#-KYrbiLo7J3IDCRdfmBP:-KYrfBp_00GEHpDgHi0-:b-896fix) ([raw file](https://github.com/criteo-cookbooks/centreon-engine/blob/5fdb12fc0dc7d1527929936dd54695c377791731/attributes/default.rb#L34)):*
<details><summary><i>Previously, jmauro wrote…</i></summary><blockquote>

Same as above
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3)*

**Comment by CharlesJUDITH** [comment]
> managed

---

Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3#-:-KYrfVt5vv1QynEcyx6R:brz1nle)*

**Comment by jmauro** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/criteo-cookbooks/centreon-engine/3)*

**Pull request merged**